### ISSUE TITLE
Erro "ModuleNotFoundError: No module named 'honeypots'" when running honeypots --setup servicename

### ISSUE THREAD

**Issue by sysfu** [qeeqbox/honeypots#70]
I'm trying to configure some services after installing honeypots. Getting this same error on all services.

Environment is pentoo-linux with python 3.11

```
$ honeypots --setup vnc
[INFO] For updates, check https://github.com/qeeqbox/honeypots
[WARNING] Using system or well-known ports requires higher privileges (E.g. sudo -E)
[INFO] Use [Enter] to exit or python3 -m honeypots --kill
[INFO] Parsing honeypot [normal]
{"action": "process", "dest_ip": "0.0.0.0", "dest_port": "33183", "password": "test", "server": "vnc_server", "src_ip": "0.0.0.0", "src_port": "33183", "status": "success", "timestamp": "2024-06-12T15:46:43.017490", "username": "test"}
[INFO] servers vnc running...
[INFO] Everything looks good!
Traceback (most recent call last):
  File "/usr/lib/python3.11/site-packages/honeypots/vnc_server.py", line 26, in <module>
    from honeypots.helper import (
ModuleNotFoundError: No module named 'honeypots'

[INFO] [x] Stopping servers...
[INFO] [x] Killing QVNCServer server
[INFO] [x] Please wait few seconds
[INFO] [x] Cleaning
```

**Comment by giga-a** [comment]
Hi @sysfu, I'm still working on this, I should be pushing new updates by the end of this week!

**Comment by sysfu** [comment]
Issue has been resolved as of v0.66 release. Thank you.

```
$ honeypots --setup vnc
[INFO] For updates, check https://github.com/qeeqbox/honeypots
[WARNING] Using system or well-known ports requires higher privileges (E.g. sudo -E)
[INFO] Use [Enter] to exit or python3 -m honeypots --kill
[INFO] Parsing honeypot [normal]
{"action": "process", "dest_ip": "0.0.0.0", "dest_port": "53039", "server": "vnc_server", "src_ip": "0.0.0.0", "src_port": "53039", "status": "success", "timestamp": "2024-09-26T20:40:10.804743"}
[INFO] servers vnc running...
[INFO] Everything looks good!
```
